cpp/test/phonenumbers/phonenumberutil_test.cc

```diff
@@ -1341,7 +1341,8 @@ TEST_F(PhoneNumberUtilTest, GetLengthOfGeographicalAreaCode) {
   number.set_national_number(2070313000ULL);
   EXPECT_EQ(2, phone_util_.GetLengthOfGeographicalAreaCode(number));
 
-  // A UK mobile phone, which has no area code.
+  // A mobile number in the UK does not have an area code (by default, mobile
+  // numbers do not, unless they have been added to our list of exceptions).
   number.set_country_code(44);
   number.set_national_number(7123456789ULL);
```

Comment:
Hmm, this is different from in [PhoneNumberUtilTest.java](https://github.com/googlei18n/libphonenumber/blob/master/java/libphonenumber/test/com/google/i18n/phonenumbers/PhoneNumberUtilTest.java). There GB_MOBILE has 7912345678 for the national number.

Also shouldn't PhoneNumberUtilTest.java be updated with these changes?